build/source/driver/multi_driver.f90

```diff
@@ -211,23 +212,44 @@ program multi_driver
 integer(i4b)                     :: err=0                      ! error code
 character(len=1024)              :: message=''                 ! error message
 
+! subsetting the model run
+integer(i4b)                     :: strtHRU                    ! index of the starting hru for a subset run
+character(len=10)                :: strTMP                     ! local temp string
```

Comment:
See issue #118 - in general there is no good reason to be stingy with these and we should standardize them (which is what issue 118 is about)

To me this is a case of "pre-mature optimization", which Donald Knuth has called the root of all evil in 97% of all cases (search for "premature optimization").